app/components/koi/summary_list/base.rb

```diff
@@ -40,6 +41,19 @@ def raw_value
       def inspect
         "#<#{self.class.name} #{@attribute.inspect}>"
       end
+
+      # Convenience method for rendering the content of the cell
+      # <% dl.text(:name) { |cell| tag.em(cell) } %>
+      # => <dt>Name</dt><dd><em>Jamie Banks</em></dd>
+      def to_s
+        attribute_value
+      end
```

Comment:
Confused about this method? Isn't this just overriding `to_s` method in Object class?

Reply:
Yes, that's why I added the code example. If you call a method on the cell that coerces it into a string, then this ensure that it returns the value instead of printing as `<TextComponent#12351>` or whatever. This lets you write `{ |cell| tag.em(cell) }` instead of `{ |cell| tag.em(cell.attribute_value) }`